### PR TITLE
feat(entity_tags): add an entity tag resource

### LINKED ISSUE #672: Support for NR1 Entity tags

### PR DESCRIPTION
**Pull request by ctrombley**
Resolves #672 .

### CODE REVIEW

File: build/deps.mk [hunk 1]
```diff
@@ -8,7 +8,7 @@ deps: tools deps-only
 
 deps-only:
 	@echo "=== $(PROJECT_NAME) === [ deps             ]: Installing package dependencies required by the project..."
-	@$(GO) mod tidy
+	#@$(GO) mod tidy
```
Comment (sanderblue) [inline build/deps.mk:11]: Should we change this back?

Comment (ctrombley) [reply build/deps.mk:11]: `go mod tidy` is broken with an update we made to goreleaser recently.  It's the same issue we saw in the kubernetes operator project that required indirect refs to `go-autorest`.

File: website/docs/r/entity_tags.html.markdown [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  Create and manage tags for a New Relic One entity.
+---
+
+# Resource: newrelic\_entity\_tags
+
+Use this resource to create, update, and delete tags for a New Relic One entity.
+
+A New Relic Personal API key is required to provision this resource.  Set the `api_key`
+attribute in the `provider` block or the `NEW_RELIC_API_KEY` environment
+variable with your Personal API key.
```
Comment (sanderblue) [inline website/docs/r/entity_tags.html.markdown:15]: Not a blocker, but thoughts on an additional "for more information" sentence linking to the "Configuring the Provider" page?

Comment (ctrombley) [reply website/docs/r/entity_tags.html.markdown:15]: 👍

**Pull request merged**